composer.json

```diff
@@ -1,64 +1,66 @@
 {
-    "name": "friendsofsymfony/oauth-server-bundle",
+    "name": "klapaudius/oauth-server-bundle",
```

Comment:
Submitting a PR that changes the package name is a no-go. I suspect you intended this to stay in your own fork.